packages/terra-base/src/Base.jsx

```diff
@@ -1,38 +1,74 @@
 import React, { PropTypes } from 'react';
-import './Base.scss';
+
+import { I18nProvider, i18nLoader } from 'terra-i18n';
+
+import BaseStyles from './BaseStyles';
 
 const propTypes = {
   /**
    * The component(s) that will be wrapped by `<Base />`.
    */
   children: PropTypes.node.isRequired,
+  /**
+   * Customized translations provided by consuming application.
+   */
+  customMessages: PropTypes.object,
+  /**
+   * The locale name.
+   */
+  locale: PropTypes.string,
```

Comment:
Updated in https://github.com/cerner/terra-core/pull/231/commits/8ac922bcb942e78cf1747a2af0d185f7ea7cbb69